charter.md

```diff
@@ -0,0 +1,355 @@
+<!-----
+* Docs to Markdown version 1.0β31
+* Tue Jan 04 2022 05:45:22 GMT-0800 (PST)
+* Source doc: observe-k8s.dev working doc
+* This document has images: check for >>>>>  gd2md-html alert:  inline image link in generated source and store images to your server. NOTE: Images in exported zip file from Google Docs may not appear in  the same order as they do in your doc. Please check the images!
+----->
+
+<p style="color: red; font-weight: bold">>>>>>  gd2md-html alert:  ERRORs: 0; WARNINGs: 0; ALERTS: 2.</p>
```

Comment:
Should we remove warnings/alerts from the generation process?

Reply:
Yes indeed  :)